security/openacademy_security.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="True">
```

Comment:
This tag is deprecated, just use `<odoo noupdate="1">` and you won't require an extra `<data>`